websites/R/Roblox Forum/presence.ts

```diff
@@ -0,0 +1,48 @@
+const presence = new Presence({
+	clientId: "993929577099448481",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "ig-mal",
+		smallImageKey: "small",
+		smallImageText: "Swooosh!",
```

Comment:
I have no clue what this means and it doesn't really make sense either

Reply:
idk it doesn't mean anything, i'll remove it lol

Reply:
done, removed